doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -325,57 +325,68 @@ The following table indicates the software maturity levels of the support for ea
            - nRF54L05
            - nRF54L10
            - nRF54L15
-           - nRF54LM20
+           - nRF54LM20A
+           - nRF54LV10A
          * - **Bluetooth®**
            - Supported
            - Supported
            - Supported
            - Experimental
+           - Experimental
          * - **Bluetooth Mesh**
            - Supported
            - Supported
            - Supported
            - --
```

Comment:
Could you please add Experimental support for nRF54LM20A for Bluetooth Mesh? (Or I have to create a separate PR for that)
nRF54LV10A is correct with --.